config/config_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
```

Comment:
I would suggest to use github.com/stretchr/testify in tests instead of reflect.

Reply:
Any reason to use another package?